plat/nvidia/tegra/include/tegra_private.h

```diff
@@ -34,6 +34,12 @@
 #include <xlat_tables.h>
 #include <platform_def.h>
 
+/*******************************************************************************
+ * Tegra DRAM memory base address
+ ******************************************************************************/
+#define TEGRA_DRAM_BASE		0x80000000
+#define TEGRA_DRAM_END		0x27FFFFFFF
```

Comment:
This is defining the end as the last byte - in the region bounds checking code the region end is calculated as the base plus the size - i.e. the first byte after the region. This inconsistency might result in off-by-one boundary calculations.